src/web/Navigator.tsx

```diff
@@ -137,10 +137,23 @@ export class Navigator extends RX.Navigator<NavigatorState> {
     constructor(initialProps?: Types.NavigatorProps) {
         super(initialProps);
 
+        // handle optional initialRouteStack or initialRoute props
+        let { initialRouteStack, initialRoute } = initialProps;
+        if(!Array.isArray(initialRouteStack)) {
+            initialRouteStack = [];
+        }
+
+        if(initialRouteStack.length === 0 && initialRoute) {
+            initialRouteStack.push(initialRoute);
+        }
+        
+
         // Default navigator state
         this.state = {
             sceneConfigStack: [],
-            routeStack: [],
+            presentedIndex: 0,
```

Comment:
Without this the navigator fails to transition for me because of the code at line 428. I'm open to an alternative fix for this if you would prefer such as changing the if statement to `!= null` etc.

```
// undefined !== null so this block is entered when it should not be
if (this.state.transitionFromIndex !== null) {                         
```